types/recharts/index.d.ts

```diff
@@ -681,7 +682,7 @@ export interface TreemapProps {
 	aspectRatio: number;
 	isAnimationActive?: boolean;
 	animationBegin?: number;
-	animationDuration?: number;
+	// animationDuration?: number;
```

Comment:
If the property is not there any more, could you please *remove* it instead of commenting it out?